actions/class.ResultServerStateFull.php

```diff
@@ -74,70 +71,6 @@ protected function returnSuccess($rawData = "") {
         exit(0);
     }
 
-    /**
```

Comment:
https://oat-sa.atlassian.net/browse/TAO-6784
spike with methods that need to be deleted, but I guess that it could be done as a separate task to not mix them (task with annotation and deletion)
And here I have not found any usages of these methods